plugins/contrast/index.js

```diff
@@ -6,6 +6,7 @@
 let $ = require("jquery");
 let Plugin = require("../base");
 let annotate = require("../shared/annotate")("labels");
+let blinder = require("../../node_modules/color-blind/lib/blind").Blind;
```

Comment:
Fancy! Updated.

I can't use only `"color-blind"` as I don't use the index js from the package, but I updated it to `"color-blind/lib/blind"`.